test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/ComprehensiveDeferQueryWithDifferentServiceCalls.kt

```diff
@@ -0,0 +1,119 @@
+package graphql.nadel.tests.next.fixtures.defer
+
+import graphql.nadel.NadelExecutionHints
+import graphql.nadel.tests.next.NadelIntegrationTest
+import kotlin.test.Ignore
+
+@Ignore
```

Comment:
ignored the tests with multiple service calls for now, until the fix for flakey `delayedResults` list order is merged

Reply:
(removed the @ignore as https://github.com/atlassian-labs/nadel/pull/556 has been merged now